composer.json

```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^7.2 || ^8.0",
```

Comment:
Maybe we can make this more explicit. If PHP 8.2 arrives, we don't know if we support this out of the box. So I would suggest: `"php": "^7.2 || ~8.0.0 || ~8.1.0"`,

Reply:
According to my understanding php will not break for 8.x versions. In case that will be a bug.

Reply:
Isn't it CI testing that reveals which PHP versions are tested and supported?

https://github.com/auraphp/Aura.Di/blob/4.x/.github/workflows/ci.yml#L38